pager/src/main/java/com/google/accompanist/pager/PagerIndicator.kt

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2021 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.accompanist.pager
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.offset
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.Shape
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * Contains default values used for [PagerIndicator].
+ */
+object PagerDefaults {
+    /**
+     * Default spacing used for [PagerIndicator].
+     */
+    val Spacing = 8.dp
+
+    /**
+     * Default indicator size used for [PagerIndicator].
+     */
+    val Size = 8.dp
+
+    /**
+     * Default shape used for [PagerIndicator].
+     */
+    val Shape = CircleShape
+
+    /**
+     * Default Indicator color value used for [PagerIndicator].
+     */
+    val IndicatorColor = Color.White
```

Comment:
Should it be replaced with `LocalContentColor.current`? White is not friendly under the light theme

Reply:
I originally Planned something like this and thought of both LocalContentColor or directly MaterialTheme. The issue is that both solutions rely on the material dependency wich is currently not part of the pager module and i did not want to pull that dependency in. I think there is a discussion needed for that and maybe the indicator needs to become a seperate module.

Reply:
Yep, this is one of the reasons why I was thinking of extracting the indicator to a different module `pager-indicators` or something. We could make that depend on `compose-material`. I'll do it quickly now.

Reply:
Do you think that it might make sense to have a generic pager indicator like the current ones and preconfigured ones like a DotPagerIndicator ? I could create another pr for that.

I would in this case remove the default shape from the defaults so that indicator always requires a shape with size and DotPagerIndicator just accepts the size and calls PagerIndicator with the CircleShape